scipy/io/_fortran.py

```diff
@@ -127,55 +156,103 @@ def read_record(self, dtype=None):
 
         Notes
         -----
-        If the record contains a multi-dimensional array, calling reshape or
-        resize will restructure the array to the correct size.
-        Since Fortran multidimensional arrays are stored in column-major format,
-        this may have some non-intuitive consequences. If the variable was
-        declared as 'INTEGER var(5,4)', for example, var could be read with
-        'read_record(dtype=np.integer).reshape( (4,5) )' since Python uses
-        row-major ordering of indices.
+        If the record contains a multi-dimensional array, you can specify
+        the size in the dtype. For example::
+
+            INTEGER var(5,4)
 
-        One can transpose to obtain the indices in the same order as in Fortran.
+        can be read with::
+
+            read_record('(4,5)i4')
+
+        Note that the fact that the array is laid out in Fortran column major
```

Comment:
As I understand, in "the fact [that]" the "that" is usually present in formal writing, so this shouldn't be wrong as it is. OTOH, maybe you are a native speaker.